libraries/streaming-e2e-tests/tests/streaming-e2e.test.js

```diff
@@ -0,0 +1,102 @@
+const path = require('path');
+const dotenv = require('dotenv');
+const ENV_FILE = path.join(__dirname, '.env');
+dotenv.config({ path: ENV_FILE });
+
+const assert = require('assert');
+const { Builder, By, Condition, Key, until, css } = require('selenium-webdriver');
+const { Options } = require('selenium-webdriver/chrome');
+
+const pathToChromeDriver = process.env.CHROME_DRIVER;
+
+describe('Chrome', function () {
+  it('should receive an echo after sending a message', function () {
+    (async () => {
+      const driver = createDriver('chrome', pathToChromeDriver);
+      
+      await echoMessageInBrowser(driver);
+      
+      const transcriptMessages = await getTranscriptMessages(driver, 3);
+      const expectedBotWelcomeMessage = 'Hello and welcome!';
+      const expectedUserMessage = 'Why hello there';
+      const expectedBotEcho = 'Streaming Echo: Why hello there.';
+
+      assert.strictEqual(transcriptMessages[0], expectedBotWelcomeMessage);
+      assert.strictEqual(transcriptMessages[1], expectedUserMessage);
+      assert.strictEqual(transcriptMessages[2], expectedBotEcho);
+      assert.strictEqual(transcriptMessages.length, 3);
+
+      driver.quit();
+    })();
+  });
+});
+
+function createDriver(browser, driverBinPath) {
+    // For now, we are only using ChromeDriver
+    // In future expansions on E2E streaming tests, we can expand to create options for multiple browsers
+    const options = new Options().setChromeBinaryPath(driverBinPath);
+    const builder = new Builder()
+        .setChromeOptions(options)
+        .forBrowser(browser)
+        .build();
+
+  return builder;
+}
+
+async function echoMessageInBrowser(driver) {
+  try {
+    await driver.get(process.env.REACT_APP_ENDPOINT);
```

Comment:
May be you can consider hosting the server locally, so we don't need to ZipDeploy to Azure Web App, and then hosting the server publicly.

Reply:
Agreed!

Per PR review meeting yesterday, originally published React App, as Steven and I weren't sure we would get automated browser testing in on time for R10, so we deployed React App to at least be able to publish manually

Created [issue#2607](https://github.com/microsoft/botbuilder-js/issues/2607) to implement locally-hosted React App for this pipeline for R11 (not enough time for R10)

Reply:
Whoops, now that's a dupe. Steven beat me to it!

Here's [the actual issue then](https://github.com/microsoft/botbuilder-js/issues/2603), and I'll close the duplicate 😆